catalog/resource_sql_table_test.go

```diff
@@ -587,6 +588,176 @@ func TestResourceSqlTableUpdateView_Comments(t *testing.T) {
 	assert.Equal(t, "barview", d.Get("name"))
 }
 
+func TestResourceSqlTableUpdateTable_Columns(t *testing.T) {
```

Comment:
Please add a unit test covering change in nullability.
I'd also like to see an example where multiple changes in a single column are requested.